### PR TITLE
Resprites glass windows and windoors, makes them 3/4ths compliant

### PR DESCRIPTION
**Pull request by RipGrayson**
## About The Pull Request

Title.
I did the windows myself, the windoor sprites are by https://github.com/aGhostCoder, with some minor adjustments.
I also adjusted some maps because the new sprites covered things like vendors, which weren't originally placed with the expectation of half the sprite being obscured by a glass wall.

## Why It's Good For The Game

No longer will TGMC have to feel envy when we look at CM and see their shiny 3/4ths wall sprites, no longer will other servers look down and laugh at us for not having correct perspective on our wall objects! I will personally strangle any and all incorrect perspective sprites so hard their original spriters will feel my fury from 1000 servers away!

## Changelog
:cl:
imageadd: Created new versions of the glass window sprites, implements some windoor sprites from another artist (see the PR for details).
imagedel: Removed some old glass window sprites.
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
The soul....

**Comment by RipGrayson** [comment]
> The soul....

Soul is secondary to eliminating the 1 dimensional eyesores this codebase got from CM.

**Comment by Kuro020** [comment]
> The soul....

I hate soul

**Review by Kuro020 — APPROVED** [review]

File: code/game/objects/machinery/doors/windowdoor.dm [hunk 1]
```diff
@@ -25,6 +25,9 @@
 
 /obj/machinery/door/window/Initialize(mapload, set_dir)
 	. = ..()
+	if(dir == 1)
```
Comment (TiviPlus) [inline code/game/objects/machinery/doors/windowdoor.dm:28]: use NORTH pls

File: code/game/objects/structures/window.dm [hunk 2]
```diff
@@ -305,6 +305,32 @@
 	reinf = TRUE
 	explosion_block = EXPLOSION_BLOCK_PROC
 	real_explosion_block = 2
+	layer = ABOVE_MOB_LAYER
+	//are we tinted or not
+	var/tinted = FALSE
```
Comment (TiviPlus) [inline code/game/objects/structures/window.dm:310]:
```suggestion
	///are we tinted or not
	var/tinted = FALSE
```

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -305,6 +305,32 @@
...
+
+/obj/structure/window/reinforced/Initialize(mapload)
+	. = ..()
+	if(dir == 1)
```
Comment (TiviPlus) [inline code/game/objects/structures/window.dm:314]: ditto

File: code/game/objects/structures/window.dm [hunk 4]
```diff
@@ -305,6 +305,32 @@
...
+		add_overlay(image(icon, "rwindow_overlay", layer = WINDOW_LAYER))
+		layer = WINDOW_FRAME_LAYER
+	if(dir == 8 || dir == 4)
```
Comment (TiviPlus) [inline code/game/objects/structures/window.dm:317]: ok in general please change all the dir checks to use the defines pls

File: code/game/objects/structures/window.dm [hunk 5]
```diff
@@ -305,6 +305,32 @@
...
+			return
+		if(locate(/obj/structure) in adj)
+			return
+		if(locate(/obj/machinery) in adj)
+			return
+		else
+			if(tinted)
+				add_overlay(image(icon, "twindowstake", layer = ABOVE_ALL_MOB_LAYER))
+				return
+			add_overlay(image(icon, "windowstake", layer = ABOVE_ALL_MOB_LAYER))
```
Comment (TiviPlus) [inline code/game/objects/structures/window.dm:329]:
```suggestion
		if(tinted)
			add_overlay(image(icon, "twindowstake", layer = ABOVE_ALL_MOB_LAYER))
			return
		add_overlay(image(icon, "windowstake", layer = ABOVE_ALL_MOB_LAYER))
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/doors/windowdoor.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/machinery/doors/windowdoor.dm:28]: still pending

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**